lib_xua/src/core/user/audiohw/audiohw.c

```diff
@@ -9,6 +9,12 @@ void AudioHwInit()
     return;
 }
 
+void AudioHwDeInit() __attribute__ ((weak));
+void AudioHwDeInit()
```

Comment:
I'm not a massive fan of the term "DeInit" but I generally try and avoid debates on naming ;) PowerDown, Disable, Shutdown etc seem more appropriate to the functionality we want.

Reply:
Ha - I think I got this from our Boston friends! Gone with AudioHwShutdown